visualizer/style/exampleGenerationLinks.css

```diff
@@ -0,0 +1,61 @@
+.currentExampleLink.coveredExampleLink {
+  text-decoration: underline;
+  color: #268BD2;
+  cursor: pointer;
+  font-weight: bold;
+}
+
+.coveredExampleLink {
```

Comment:
Looks like there are some redundant rules here, with the `.currentExampleLink.coveredExampleLink` block above. Or am I wrong?